lib/removeUnusedSkins.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removeUnusedSkins;
+
+function removeUnusedSkins(gltf, stats) {
+    var usedSkinIds = {};
+    var nodes = gltf.nodes;
+
+    // Build hash of used skins by iterating through nodes
+    if (defined(nodes)) {
+        for (var nodeId in nodes) {
+            if (nodes.hasOwnProperty(nodeId)) {
+                if(nodes[nodeId].hasOwnProperty("skin")){
+                    var id = nodes[nodeId].skin;
+                    usedSkinIds[id] = true;
+                }
+            }
+        }
+    }
+
+    // Iterate through skins and remove those that are not in the hash
+    var numberOfSkinsRemoved = 0;
+    var skins = gltf.skins;
+    if (defined(skins)) {
+        var usedSkins = {};
+
+        for (var skinId in skins) {
+            if (skins.hasOwnProperty(skinId)) {
+                // If this skin is in the hash, then keep it in the glTF asset
+                if (defined(usedSkinIds[skinId])) {
+                    usedSkins[skinId] = skins[skinId];
+                } else {
+                    ++numberOfSkinsRemoved;
+                }
+            }
+        }
+
+        if (defined(stats)) {
+            stats.numberOfSkinsRemoved += numberOfSkinsRemoved;
+        }
+
+        gltf.skins = usedSkins;
+    }
+
+// TODO: remove orphan uris
```

Comment:
Remove this `TODO` since a skin doesn't directly have a uri.  However, we will want to remove skins before accessors, since a skin's `inverseBindMatrices` property points to an `accessor`, removing an unused `skin` will likely create an unused `accessor`.